### PR TITLE
Improve docker-compose script to handle docker compose and docker-compose.

### PR DESCRIPTION
**Pull request by corneil**
Related to #5582.

### CODE REVIEW

File: src/docker-compose/dc.sh [hunk 1]
```diff
@@ -36,6 +38,16 @@ while [ "$1" != "" ]; do
     "postgres" | "postgresql")
         DATABASE=postgres
         ;;
+    "--no-wait")
+        NO_WAIT=true
+        ;;
+    "down" | "up" | "rm" | "kill" | "stop" | "start" | "run" | "restart" | "pull" | "pause" | "create" | "build")
+        if [ "$DC_CMD" != "" ];then
```
Comment (onobc) [inline src/docker-compose/dc.sh:45]: Is `DC_CMD` newly added here? Should it be in the usage?

File: src/docker-compose/dc.sh [hunk 2]
```diff
@@ -6,19 +6,21 @@ fi
...
 SCDIR=$(realpath $SCDIR)
 if [ "$1" == "" ]; then
-    echo "Usage is: "${BASH_SOURCE[0]}" <database> <broker> [options] [flags]"
+    echo "Usage is: "${BASH_SOURCE[0]}" <database> <broker> [options] [flags] [--no-wait]"
     echo "Where:"
     echo "    database: is one of mariadb, mysql or postgres"
     echo "    broker: is on of kafka or rabbitmq"
     echo "    options: one or more of ssl, zipkin, prometheus, influxdb, wavefront, ssl, dood, debug-dataflow, debug-skipper"
     echo "    flags: any docker-compose options"
+    echo "    --no-wait: Don't display command line and don't wait for input."
```
Comment (onobc) [inline src/docker-compose/dc.sh:15]: Should we say what the default is?

**Review by onobc — CHANGES_REQUESTED** [review]
@corneil the changes LGTM but do you mind updating the PR description to indicate how this fixes #5582? Also, you may want to move the commit to "See #5582" since there is another aspect of that issue - to check all of the docker compose scripts (unless you already did that and do in fact want the issue to be closed w/ this PR). Thanks

File: src/docker-compose/dc.sh (hunk 2 above)
Comment (corneil) [reply src/docker-compose/dc.sh]: I decided that waiting for input on all executions is problematic and switched it around by adding a --wait option instead.

**Comment by corneil** [comment]
> @corneil the changes LGTM but do you mind updating the PR description to indicate how this fixes #5582? Also, you may want to move the commit to "See #5582" since there is another aspect of that issue - to check all of the docker compose scripts (unless you already did that and do in fact want the issue to be closed w/ this PR). Thanks

This doesn't resolve the issue, it is only related.
The real issue is the the local launcher is expecting output files for stdout and stderr to exist in the dataflow or skipper container when they are on the host or other container filesystem.

File: src/docker-compose/dc.sh (hunk 1 above)
Comment (onobc) [reply src/docker-compose/dc.sh:46]: I don't see how to use the script in the usage (the DC_CMD) anyways. Maybe I am missing something.

**Review by onobc — APPROVED** [review]
LGTM @corneil - nice work!

**Pull request merged**